cmc-domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/ccj/RepaymentPlan.java

```diff
@@ -1,16 +1,17 @@
-package uk.gov.hmcts.cmc.claimstore.models.ccj;
+package uk.gov.hmcts.cmc.domain.models.ccj;
 
 import org.apache.commons.lang3.builder.ReflectionToStringBuilder;
-import uk.gov.hmcts.cmc.claimstore.constraints.DateNotInThePast;
-import uk.gov.hmcts.cmc.claimstore.constraints.Money;
+import uk.gov.hmcts.cmc.domain.models.constraints.DateNotInThePast;
+import uk.gov.hmcts.cmc.domain.models.constraints.Money;
+import uk.gov.hmcts.cmc.domain.utils.ToStringStyle;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'uk.gov.hmcts.cmc.domain.utils.ToStringStyle'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.reform.hmcts.net/coding_rules#rule_key=squid%3AUselessImportCheck)